src/test/java/fi/metatavu/kuntaapi/test/server/integration/ptv/AbstractPtvTest.java

```diff
@@ -0,0 +1,98 @@
+package fi.metatavu.kuntaapi.test.server.integration.ptv;
+
+import static org.awaitility.Awaitility.await;
+
+import java.util.concurrent.TimeUnit;
+
+import org.skyscreamer.jsonassert.Customization;
+
+import com.jayway.restassured.http.ContentType;
+import com.jayway.restassured.path.json.JsonPath;
+
+import fi.metatavu.kuntaapi.test.AbstractIntegrationTest;
+import fi.metatavu.kuntaapi.test.json.JSONAssertCustomizations;
+
+/**
+ * Abstract base class for all PTV tests
+ * 
+ * @author Antti Leppä
+ */
+public abstract class AbstractPtvTest extends AbstractIntegrationTest {
+  
+  /**
+   * Waits that all organization services have been discovered
+   *  
+   * @param organizationIndex organization index
+   */
+  protected void waitOrganizationServices(int organizationIndex) {
+    String organizationId = getOrganizationId(organizationIndex);
+    
+    await().atMost(5, TimeUnit.MINUTES).until(() -> {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove useless curly braces around statement and then remove useless return keyword [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1602)